Visited Places/introduce.html

```diff
@@ -29,7 +29,7 @@
 
 <body>
     <header>
-        <h1>The places i visited in USA</h1>
+        <h1>The places we visited in USA</h1>
```

Comment:
Why did you change this? This does not make sens.

Reply:
For cleaning up.